ServerCore/Pages/Resources/nipd2023/RulesPartial.cshtml

```diff
@@ -0,0 +1,87 @@
+                    <li>giving false information to another team</li>
+                    <li>modifying something needed by other teams to solve a puzzle</li>
+                    <li>trying to purloin answers from another team, and</li>
+                    <li>colluding with other teams</li>
+                </ul>
+            </li>
+            <li>In order for a puzzle to be marked as solved, the correct answer must be entered into the correct answer submission box on the website and be submitted before the event ends.</li>
+            <li>Each team must be reachable by email during the event since new puzzles, instructions, and corrections to puzzles may need to be sent out via email. <b>Make sure mail from msph@puzzlehunt.org isn't going to junk!</b></li>
+            <li>Teams should clean up their conference rooms after the end of the event.  The regular cleaning schedule does not cover weekends, so you should leave the room in an appropriate condition to be used for a Monday morning meeting.</li>
+        </ol>
+        <h3>Traveling outside your conference room</h3>
+        <ol>
+            <li>
+                There are only two ways you may be instructed to travel:
+                <ul>
+                    <li>The puzzle has explicit instructions to travel. For example, your team may need to attend a live event at a specified location.</li>
+                    <li>You submit an answer and the response from the website gives you explicit instructions to travel.  If you have an answer that looks like a travel instruction, enter it into the submission box in full and only travel if the response says so.</li>
+                </ul>
+            </li>
+            <li>You must not enter any person's private office during the course of the event unless it is the office of a member of your team.</li>
+            <li>If any travel is required, it will always be to Building 99.  Additionally, Studios B, C, X, Buildings 92, 112, 114, 120, 123 - 127, 25, 34, 35, 41, 42, 87, and all construction areas are OFF LIMITS and should not be entered unless your personal office is in them.</li>
+        </ol>
+        <h3>Team registration</h3>
+        <p>Teams can have a maximum of 6 people, and we heavily encourage filling up your team as much as possible. You may register your team with only 1 person and fill out the roster later.</p>
+        <ul>
+            <li>The event is designed for Microsoft FTEs, their family, and friends. All guests must be registered as a member of your team and will count against your limit of 6 people.</li>
+            <li>Microsoft interns are not permitted to participate in this specific event.</li>
+            <li>Teams must have a minimum of 2 FTEs.  There are no requirements to be on campus, and there are no restrictions for guests being on campus, although keep in mind that guests will need to be accompanied by FTEs so they can get into buildings.</li>
```

Comment:
"no requirements" isn't entirely accurate -- it's where opening, closing, and live events are

Reply:
That was more about remote participation, so I'll change it to explicitly say remote participation is allowed.